RocketChat/Rocket.Chat.ReactNative
# [FIX] Share Extension shows a server that has no user logged in

## Proposed changes


## Issue(s)	


## How to test or reproduce

1 - Start the process of logging in to a server by choosing one, but do not log in to any user account.
       a - If this step is not done correctly, at step 4, the user will probably see the screen stating there is no server logged in.
2 - Minimize app
3 - Open Share Extension (by sharing something from the device's gallery)
4 - Try choosing a server to share the file to.

## Screenshots

#### Before
https://user-images.githubusercontent.com/47038980/172525539-58c63246-0fb5-488f-a674-0edfaf84ad0e.mov



#### After
https://user-images.githubusercontent.com/47038980/172525273-174297ca-1746-41a1-861d-7d150e46faa3.mov

## Types of changes




## Checklist



## Further comments
